ext/nnstreamer/tensor_converter/tensor_converter_python3.cc

```diff
@@ -212,19 +212,25 @@ PYConverterCore::convert (GstBuffer *in_buf, GstTensorsConfig *config)
 int
 PYConverterCore::init ()
 {
+  int ret = 0;
   /** Find nnstreamer_api module */
   PyObject *api_module = PyImport_ImportModule ("nnstreamer_python");
   if (api_module == NULL) {
-    return -EINVAL;
+    ret = -EINVAL;
+    goto done;
```

Comment:
Wouldn't "early return", code before using "goto" in general, be desirable?
It seems to be the same function. Is there a reason for the change?

Reply:
Thanks. That was a residue from older changes.